.eslintrc

```diff
@@ -119,13 +119,11 @@
     "import/no-deprecated": "error",
     "import/no-duplicates": "error",
     "import/no-dynamic-require": "error",
-    "import/no-extraneous-dependencies": ["error", { "packageDir": ["./", "./common/ts"] }],
```

Comment:
Not necessary, because TS already throws an error when a dependency types are not resolved 

Currently produces such false positives
![image](https://github.com/SonarSource/sonar-scanner-vsts/assets/31401273/4585e551-2247-46a9-9eee-a9b230cef4f7)